package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "playground",
+  "private": true,
+  "version": "0.0.1",
+  "type": "module",
+  "scripts": {
+    "build": "npm run build --prefix packages/@orchidui-vue && npm run build-docs && npm run build-playground",
+    "dev": "npm run dev --prefix packages/@orchidui-vue",
+    "dev-playground": "vite",
+    "build-playground": "vite build",
```

Comment:
```suggestion
    "build": "npm run build --prefix packages/@orchidui-vue && npm run build-docs && npm run build-production",
    "dev": "npm run dev --prefix packages/@orchidui-vue",
    "dev-playground": "vite",
    "build-production": "vite build",
```

Reply:
@orchid-vue/public/icons & @orchid-icon/public/icons should we remove in @orchid-vue/public/icons : 

we have issues here : npx command to import icons : ref @orchidui-icons [reference https://ui.shadcn.com/ ]

Reply:
cli command updated